.travis.yml

```diff
@@ -168,3 +168,19 @@ before_install:
   - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew install crystal-lang; fi
   # Bash
   - pip install beautysh
+  # Elixir
+  - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then
+      brew install elixir;
+    fi
+  - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then
+      wget https://packages.erlang-solutions.com/erlang-solutions_1.0_all.deb;
```

Comment:
If you were feeling really ambitious and had experience with Docker, you could contribute a `mix` and/or `exfmt` Docker image to https://github.com/Unibeautify/docker-beautifiers . However, I do not require it. Just a suggestion 😄 .